js/src/modals/Transfer/transfer.js

```diff
@@ -86,12 +85,12 @@ class Transfer extends Component {
   render () {
     const { stage, extras } = this.state;
 
+    // title={ this.renderAccount() }
```

Comment:
Not yet. It may need to come back since it is the account in the header - not quite sure yet if the icon on the button only is sufficient.